### PR TITLE
FWF-3794:[Feature] - Import in Create or Edit page

### PR DESCRIPTION
**Pull request by Josephalexantony-aot**
# Issue Tracking

JIRA: https://aottech.atlassian.net/browse/FWF-3794
Issue Type: FEATURE

# Screenshots (if applicable)
image
image
image
image
image
image
image

### CODE REVIEW

File: forms-flow-web/src/components/Form/EditForm/FormEdit.js [hunk 1]
```diff
@@ -101,7 +105,98 @@ const Edit = React.memo(() => {
...
+      setFormSubmitted(true);
+      // Handle selectedLayoutVersion logic
+      if (selectedLayoutVersion || selectedFlowVersion) {
+        data.form = {
+          skip: selectedLayoutVersion === true,
+        };
 
+        data.form = {
+          skip: selectedFlowVersion === true,
+        };
```
Comment (arun-s-aot) [inline forms-flow-web/src/components/Form/EditForm/FormEdit.js:153]: There is repetition in assignment here

Comment (Josephalexantony-aot) [reply forms-flow-web/src/components/Form/EditForm/FormEdit.js]: Payload updated.

**Review by abilpraju-aot — APPROVED** [review]

File: forms-flow-web/src/components/Modals/ImportModal.js [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+    ];
+
+    const flowOptions = [
+        { value: true, label: 'Skip, do not import' },
+        { value: 'major', label: `import as version ${fileItems?.workflow?.majorVersion ?? 1}.${fileItems?.workflow?.minorVersion ?? 0} (only impacts new submissions)` }
+    ];
+
+    const handleLayoutChange = (option) => {
+        setSelectedLayoutOption(option);
+        console.log("Selected Layout:", option.value);
```
Comment (shuhaib-aot) [inline forms-flow-web/src/components/Modals/ImportModal.js:32]: remove console

File: forms-flow-web/src/components/Modals/ImportModal.js [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+    };
+    const handFlowChange = (option) => {
+        setSelectedFlowOption(option);
+        console.log("Selected Layout:", option.value);
```
Comment (shuhaib-aot) [inline forms-flow-web/src/components/Modals/ImportModal.js:36]: remove console

**Review by shuhaib-aot — APPROVED** [review]

**Comment by arun-s-aot** [comment]
Merging the PR. SonarCloud issues can be resolved as part of next ticket https://aottech.atlassian.net/browse/FWF-3834 as we are optiising the code @Josephalexantony-aot

**Pull request merged**